stock_cancel/AUTHORS.txt

```diff
@@ -0,0 +1,2 @@
+Andrea Cometa (<http://www.andreacometa.it>)
+Agile Business Group sagl (<http://www.agilebg.com>)
```

Comment:
Please put this in a README.rst file with the description field.
You should consider to use https://github.com/OCA/maintainers-tools/blob/master/template/module/README.rst as template.